PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/helpers/IDEHelperImpl.java

```diff
@@ -75,11 +80,18 @@
 import javax.swing.*;
 import java.io.*;
 import java.nio.charset.StandardCharsets;
+import java.time.ZonedDateTime;
 import java.util.*;
 import java.util.concurrent.ExecutionException;
 
 @Log
 public class IDEHelperImpl implements IDEHelper {
+
+    private static final String APP_SERVICE_FILE_EDITING = "App Service File Editing";
+    private static final String FILE_HAS_BEEN_DELETED = "File %s has been deleted from Azure, do you still want to save your changes?";
```

Comment:
from Azure -> from remote server, do you want to create a new file with the changed content?

Reply:
File '%s'